go.mod

```
@@ -0,0 +1,3 @@
+module github.com/jbowens/dictionary
+
+go 1.12
```

any particular reason for 1.12?

```suggestion
go 1.13
```